ArcMovies/Helpers/ColorHelper.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ColorHelper.swift
+//  ArcMovies
+//
+//  Created by Jaime Costa Marques on 13/05/18.
+//  Copyright © 2018 Jaime Costa Marques. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
+    
+    func lighter(by percentage: CGFloat = 30.0) -> UIColor? {
+        return self.adjust(by: abs(percentage) )
```

Comment:
Why did you added `self.` here?

Reply:
Not necessary I need the self here. 
I just typed it to fire the autocomplete, although I could use ^ Space.